FIRE_DATA_API/fire_data_api/functions.py

```diff
@@ -101,4 +105,33 @@ def sort_fires(values):
         else:
             other_fires.append(fire)
 
-    return nearby_fires, other_fires
\ No newline at end of file
+    return nearby_fires, other_fires
+
+#Get the air quality data from waqi.info
+def get_aqi_data(latitude,longitude):
+    base_url = " https://api.waqi.info/feed/geo"
+    api_url = f'{base_url}:{latitude};{longitude}/?token={TOKEN}'
+    response = requests.get(api_url)
+    recd_data = response.json()
+    if(recd_data['status']=='ok'):
+        #Retrieving only the air quality data
+        aq_data = recd_data['data']['iaqi']
+    else:
+        aq_data = recd_data['data']
+
+    return aq_data
+
+#Find the co-ordinates of nearest weather stations
+def get_nearest_stations(latitude,longitude,distance):
+    base_url = " https://api.waqi.info/map/bounds/?latlng="
+    #Getting the bounding box for given co-ordinates
```

Comment:
What do you mean by "bounding box"? Do you mean the perimeter for given coordinates? Please explain. Thanks.

Reply:
By bounding box , I meant creating a box with perimeter containing all co-ordinates which are less than the distance range of the given co-ordinates